fire/cli/mtl.py

```diff
@@ -446,14 +420,31 @@ def opbyg_punktoversigt(
         if punkt in nye_punkter:
             continue
 
-        info = punkt_information(punkt)
-        kote = punkt_kote(info, koteid)
-        if kote is not None:
-            punktoversigt.at[punkt, "kote"] = kote.z
-            punktoversigt.at[punkt, "σ"] = kote.sz
-            punktoversigt.at[punkt, "år"] = kote.registreringfra.year
+        fire.cli.print(f"Finder kote for {punkt}", fg="green")
+        pkt = firedb.hent_punkt(punkt)
+
+        # Grav aktuel kote frem
+        kote = None
+        for koord in pkt.koordinater:
+            if koord.sridid != DVR90.sridid:
+                continue
+            if koord.registreringtil is None:
+                kote = koord
+                break
+        if kote is None:
+            fire.cli.print(
+                f"Ingen aktuel DVR90-kote fundet for {punkt}",
+                bg="red",
+                fg="white",
+                err=True,
+            )
+            sys.exit(1)
 
-        geom = punkt_geometri(info, punkt)
+        punktoversigt.at[punkt, "kote"] = kote.z
+        punktoversigt.at[punkt, "σ"] = kote.sz
+        punktoversigt.at[punkt, "år"] = kote.registreringfra.year
+
+        geom = punkt_geometri(pkt)
         if pd.isna(punktoversigt.at[punkt, "φ"]):
             punktoversigt.at[punkt, "φ"] = geom[1]
             punktoversigt.at[punkt, "λ"] = geom[0]
```

Comment:
Hvad med istedet at gøre sådan her:

```suggestion
        (lon, lat)= punkt_geometri(pkt)
        if pd.isna(punktoversigt.at[punkt, "φ"]):
            punktoversigt.at[punkt, "φ"] = lat
            punktoversigt.at[punkt, "λ"] = lon
```

Reply:
Ja - det ser attraktivt og Pythonisk ud. Inlineudpakning er først dukket op i C-verdenen med C++20, så jeg skal lige vænne mig til de muligheder, der har været i Python (og Matlab og IDL o.m.a.) siden 1990erne :-)